e2e/tests-dfx/identity_command.bash

```diff
@@ -132,6 +132,17 @@ teardown() {
     assert_command_fail dfx identity remove charlie
 }
 
+@test "identity remove: only remove identities with configured wallet if --drop-wallets is specified" {
+    WALLET="rwlgt-iiaaa-aaaaa-aaaaa-cai"
+    assert_command dfx identity new --disable-encryption alice
+    assert_command dfx identity use alice
+    assert_command dfx identity --network ic set-wallet --force "$WALLET"
+    assert_command dfx identity use default
+    assert_command_fail dfx identity remove alice
+    assert_match "$WALLET"
+    assert_command dfx identity remove alice --drop-wallets
+}
+
```

Comment:
Would you please add a test that demonstrates that `dfx identity remove` displays the wallet ids when removing a persistent identity?

You can configure the local network to be persistent for this case.  Here's a snippet from another test that does this:
```
    # shellcheck disable=SC2094
    cat <<<"$(jq .networks.local.type=\"persistent\" dfx.json)" >dfx.json
```

Reply:
On line 139 `set-wallet --force` adds the wallet for a persistent network and line 142 checks that the wallet is listed there. I made it bit more clear what the test is actually checking for.

I'd like to avoid the `setup_actuallylocal_network` because that would make the test run for a much longer time. That's why I'm using the `--force` flag. Does this sufficiently test the functionality?

Reply:
Yes, that makes sense.